iotedgedev/cli.py

```diff
@@ -248,6 +249,38 @@ def deploy(manifest_file):
 main.add_command(deploy)
 
 
+@solution.command(
+    name="deploy-layered",
+    context_settings=CONTEXT_SETTINGS,
+    help="Create layered deployment in IoT Hub")
+@click.option("--file",
+              "-f",
+              "manifest_file",
+              default=envvars.DEPLOYMENT_CONFIG_FILE_PATH,
+              show_default=True,
+              required=False,
+              help="Specify the deployment manifest file")
+@click.option("--layered-deployment-name",
+              "-n",
+              "layered_deployment_name",
+              show_default=True,
```

Comment:
>show_default=True [](http://example.com/codeflow?start=14&length=17)

rather than setting the value by default to "layered_deployment_name" should we instead make it required? I worry that we can't garantee unique name by default and should force the user to think about the naming. What do you think?

Reply:
Oh yeah, I hadn't gotten to that part yet, since I wanted to write the tests first then fix it 😄